hardware_interface/include/hardware_interface/joint_state_interface.h

```diff
@@ -64,18 +140,75 @@ class JointStateHandle
     {
       throw HardwareInterfaceException("Cannot create handle '" + name + "'. Effort data pointer is null.");
     }
+
   }
 
   std::string getName() const {return name_;}
   double getPosition()  const {assert(pos_); return *pos_;}
   double getVelocity()  const {assert(vel_); return *vel_;}
   double getEffort()    const {assert(eff_); return *eff_;}
 
+  const double* getPositionPtr() const {return pos_;}
+  const double* getVelocityPtr() const {return vel_;}
+  const double* getEffortPtr()   const {return eff_;}
```

Comment:
In `actuator_state_interface.h`, all the data getters are together, followed by all the ptr getters. This file should follow suit.

Reply:
it should now